suites/pacific/rados/tier-3_rados_test-location-stretch-mode.yaml

```diff
@@ -307,13 +307,11 @@ tests:
         delete_pool: true
       desc: Test stretch Cluster mon replacement - Arbiter site
 
-# Commenting test until fix for bug : https://bugzilla.redhat.com/show_bug.cgi?id=2249962
-# New Bug: https://bugzilla.redhat.com/show_bug.cgi?id=2265116
   - test:
       name: Netsplit Scenarios data-data sites
       module: test_stretch_netsplit_scenarios.py
       polarion-id: CEPH-83574979
-      comments: Active bug - 2249962
+      comments: Active bug - 2265116
```

Comment:
This is the latest active bug for the issue : https://bugzilla.redhat.com/show_bug.cgi?id=2318975